applications/FluidDynamicsApplication/tests/cpp_tests/test_navier_stokes_wall_condition.cpp

```diff
@@ -0,0 +1,286 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+#include <iomanip> // for std::setprecision
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "includes/model_part.h"
+#include "includes/cfd_variables.h"
+
+// Application includes
+#include "fluid_dynamics_application.h"
+
+namespace Kratos::Testing {
+
+namespace
+{
+    auto CreateTestingNavierStokesWallCondition2D2N(ModelPart& rModelPart)
+    {
+        // Add required nodal variables
+        rModelPart.AddNodalSolutionStepVariable(DIVPROJ);
+        rModelPart.AddNodalSolutionStepVariable(ADVPROJ);
+        rModelPart.AddNodalSolutionStepVariable(NORMAL);
+        rModelPart.AddNodalSolutionStepVariable(VELOCITY);
+        rModelPart.AddNodalSolutionStepVariable(PRESSURE);
+        rModelPart.AddNodalSolutionStepVariable(BODY_FORCE);
+        rModelPart.AddNodalSolutionStepVariable(MESH_VELOCITY);
+        rModelPart.AddNodalSolutionStepVariable(EXTERNAL_PRESSURE);
+
+        // Set the required properties
+        // Note that the condition DENSITY is retrieved from the element one
+        auto p_properties_0 = rModelPart.CreateNewProperties(0);
+        auto p_properties_1 = rModelPart.CreateNewProperties(1);
+        p_properties_1->SetValue(DENSITY, 1000.0);
+        p_properties_1->SetValue(DYNAMIC_VISCOSITY, 1.0);
```

Comment:
In this particular case I think it makes no difference at all as there are no viscosity terms involved (only the stress coming from the parent element, which is already tested in its corresponding unit test). However, considering that we will ideally use this same function for the wall modelling tests, I'll change it in here to avoid changing the values in the upcoming PR(s).